contract/models/contract_line.py

```diff
@@ -641,12 +625,12 @@ def _insert_markers(self, first_date_invoiced, last_date_invoiced):
         name = self.name
         name = name.replace("#START#", first_date_invoiced.strftime(date_format))
         name = name.replace("#END#", last_date_invoiced.strftime(date_format))
+        locale.setlocale(locale.LC_TIME, lang.code)
```

Comment:
@pedrobaeza Doesn't those lines emphasized with try/except/finally ?

Reply:
Not sure what can fail there... Do you know it?

Reply:
I tried to change "LC_TIME" to "LC_ALL" to see if it was the problem in Odoo test fails, but it doesn't change...
I found this, but it seems an env issue: https://stackoverflow.com/questions/14547631/python-locale-error-unsupported-locale-setting

Reply:
Test fail because locale is not present in system. That's why I'd prefer the former solution

Reply:
Should I come back to your solution @rousseldenis?

Reply:
A fallback locale should exist, isn't it?

Reply:
> A fallback locale should exist, isn't it?

Sorry Pedro, but I don't know any fallback solution if not `try except`.
I guess it will be only useful to pass tests, but if someone installs this module in a system with no `locale` available, translations won't work.
@pedrobaeza @rousseldenis Let me know what I can do and I'll do it

Reply:
OK, at this point I can't enforce to use this technique, as I don't know if there's a minimum locale per system.